src/main/java/com/bravo/user/dao/specification/AbstractSpecification.java

```diff
@@ -86,9 +86,10 @@ protected void applyDateTimeFilter(
         inClause.add(s.toLowerCase());
         continue;
       }
-      final String targetValue = (isNot ? s.substring(1) : s).toLowerCase();
+      String targetValue = (isNot ? s.substring(1) : s).toLowerCase();
       Predicate predicate;
       if(isLike){
+    	targetValue = targetValue.replace("*", "%"); // "like" queries must use "%"
```

Comment:
Good catch! I didn't realize this bug existed in the code.

Reply:
Thanks.